core/src/test/java/org/apache/iceberg/util/TestBinPacking.java

```diff
@@ -195,7 +195,7 @@ public void testBinPackingLookBack() {
     return packer.packEnd(items, Integer::longValue);
   }
 
-  private <T> List<T> l(T... items) {
+  private <T> List<T> asList(T... items) {
```

Comment:
Nit: this is a bit more distracting now that it is `asList`. What about using `list` instead?